src/web/public/locales/zh-CN.json

```
@@ -70,6 +70,7 @@
     "payChannelAlipay": "支付宝",
     "paySuccess": "支付成功",
     "protocalDesc": "购买即表示已阅读并同意以下协议",
+    "virtualGoodsNotice": "您购买的商品为虚拟内容商品，购买后不支持退订、转让。",
```

```suggestion
    "virtualGoodsNotice": "你购买的商品为虚拟商品，购买后不支持退款、转让。",
```
We should never call user "您".

---

fixed